### ISSUE TITLE
Peer dependencies with git URLs made AurgurProject CI unhappy with NPM

### ISSUE THREAD

**Issue by pgebheim** [EdgeApp/edge-core-js#199]
https://github.com/EdgeApp/edge-core-js/blob/a590e8a4b6fcc880f1f7c39267d9b69883ff1f21/package.json#L107

Since this change our CI breaks on installing items with NPM. Works fine in yarn.

```
npm ERR! peer dep missing: babel-plugin-module-resolver@^2.5.0, required by eslint-import-resolver-babel-module@3.0.0
npm ERR! peer dep missing: react-native-fast-crypto@^1.5.3, required by edge-core-js@0.13.7
npm ERR! peer dep missing: react-native-randombytes@git://github.com/Airbitz/react-native-randombytes.git#40d02a5f922, required by edge-core-js@0.13.7
npm ERR! peer dep missing: react-native-tcp@git://github.com/Airbitz/react-native-tcp.git, required by edge-core-js@0.13.7
```

Not sure about you guys but putting in git deps for a release version seems... unstable.

**Comment by swansontec** [comment]
Good catch. I'll get this taken care of in the next release.

What happened was that out `peerDependencies` field was wrongly named, so NPM simply ignored it. Naming it correctly exposed these other hidden problems.

**Comment by pgebheim** [comment]
Oh! Gotcha :)

I'll check out the version whence you get those fixed!

**Comment by swansontec** [comment]
Pull request #201 should solve this.

**Comment by swansontec** [comment]
This was published in edge-core-js v0.14.0.
